save-backend/build.gradle.kts

```diff
@@ -18,10 +17,6 @@ openApi {
     outputDir.set(file(projectDir))
     outputFileName.set("backend-api-docs.json")
     waitTimeInSeconds.set(120)
-
-    tasks.named<BootRun>("bootRun") {
```

Comment:
It's was required basically for CI, which prohibits to use our default directory `/home/cnb/...`
Probably, you can add some flags for Windows, which will disable it on this OS, and not remove this at all

Reply:
will revert